operator/config/crd/kustomization.yaml

```diff
@@ -28,3 +23,10 @@ patchesStrategicMerge:
 # the following config is for teaching kustomize how to do kustomization for CRDs.
 configurations:
 - kustomizeconfig.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Should this be at the top?

Reply:
This is a result of `kustomize edit --fix`. Fixing this for better readability as suggested.